static/app/views/explore/components/schemaHintsList.tsx

```diff
@@ -386,7 +387,6 @@ const SchemaHintsContainer = styled('div')`
   flex-direction: row;
   gap: ${space(1)};
   flex-wrap: nowrap;
-  overflow: hidden;
```

Comment:
I wonder if this was there so that the row wouldn't overflow? Should it just be overflow-x: hidden instead?